wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -158,6 +158,20 @@
                 echo esc_html__('Induction Date and Time: ', 'goonj-blocks') . esc_html($slots['date']);
                 ?>
             </p>
+        <?php elseif ($slots['status'] === 'No_show') : ?>
+            <p>To schedule your Induction/Orientation please get in touch with the Goonj team on :</p>
+                <div class="contact-info">
+                    <div class="contact-item">
+                        <img src="https://goonj-crm.staging.coloredcow.com/wp-content/uploads/2024/08/email-icon.png" alt="Email Icon" class="icon">
```

Comment:
Can we not hardcode the image URL here because if we push it to prod then it won't work there, can we use any wp function here to avoid this?